### PR TITLE
The French Correction

### LINKED ISSUE #4483: French corrections / improvements 2022-10-27
Please make the following corrections to the French copy:

- "Browse opportunities" was translated wrong. Please use "parcourez les opportunités" everywhere it appears. (especially in the nav) (this might change again, sorry)

- Missing link in French for “sample tests” on the Languages profile sub-page

- "surmonte ces obstacles" should be "surmonter ces obstacles" at the top of Diversity, equity and inclusion page

- "seront utilisées" should be "seront utilisés" on both bullets on the Diversity, equity and inclusion page

- For all four definitions used in the modals ont he Diversity, equity and inclusion plage. Please go copy/paste the french definition from the statscan page (which is linked in the modal). The translations are all a bit off on those.

### PR DESCRIPTION
**Pull request by petertgiles**
# :wave: Introduction
This branch collections a number of French corrections together.
# :test_tube: Testing
Check that the acceptance criteria have been met:
>-  "Browse opportunities" was translated wrong. Please use "parcourez les opportunités" everywhere it appears. (especially in the nav)
>- Missing link in French for “sample tests” on the Languages profile sub-page
>- "surmonte ces obstacles" should be "surmonter ces obstacles" at the top of Diversity, equity and inclusion page
>- "seront utilisées" should be "seront utilisés" on both bullets on the Diversity, equity and inclusion page
>- For all four definitions used in the modals ont he Diversity, equity and inclusion plage. Please go copy/paste the french definition from the statscan page (which is linked in the modal). The translations are all a bit off on those.
# :robot: Robot Stuff
Closes #4483

### CODE REVIEW

File: frontend/talentsearch/src/js/lang/fr.json [hunk 1]
```diff
@@ -1496,24 +1496,24 @@
...
   },
   "bKvvaI": {
     "defaultMessage": "Conditions d'emploi",
     "description": "Legend for the Conditions of Employment filter checklist"
   },
-  "hz8OYx": {
-    "defaultMessage": "Les personnes ayant une déficience, y compris une déficience physique, intellectuelle, cognitive, mentale ou sensorielle, un trouble d'apprentissage ou de la communication ou une limitation fonctionnelle, de nature permanente, temporaire ou épisodique, manifeste ou non et dont l'interaction avec un obstacle nuit à la participation pleine et égale d'une personne dans la société.",
+  "y5Z2Li": {
+    "defaultMessage": "Réfère à une personne dont les activités quotidiennes sont limitées en raison d'un trouble ou d'une difficulté à accomplir certaines tâches. La seule exception à cet égard concerne les troubles du développement, le répondant qui a reçu un tel diagnostic étant considéré comme ayant une incapacité.",
     "description": "Definition of accepted ways to identify as person with a disability."
```
Comment (mnigh) [inline frontend/talentsearch/src/js/lang/fr.json:1517]:
```suggestion
    "description": "Definition of Person with a disability from the StatsCan 'Classification of Status of Disability' page."
```

**Review by mnigh — CHANGES_REQUESTED** [review]
I got a man in Poughkeepsie who wants to talk to you...

File: frontend/talentsearch/src/js/lang/fr.json (hunk 1 above)
Comment (petertgiles) [reply frontend/talentsearch/src/js/lang/fr.json]: Good catch, thanks!

Fixed: [Fix description of disabled translation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4487/commits/75860cad80a1b26e3490e5bae9b106c257855d53)

**Review by mnigh — APPROVED** [review]
Nice work! 👍 🚀

**Pull request merged**